tests/process/test_workflow_lines.py

```diff
@@ -0,0 +1,28 @@
+import pandas as pd
+from unittest import TestCase
+import pickle
+
+from cruisetrack.process.workflow_lines import process_lines
+
+
+class TestLineWorkflow(TestCase):
+
+    def setUp(self) -> None:
+        self.df = pd.read_csv('../data/line_layer_as_df.csv')
+        self.params1 = pickle.load(open('/home/markus/scripting/cruise_track/tests/data/line_layer_as_df.pickle', "rb"))
```

Comment:
unpickle the pickle file. returns a dictionary with all the parameters previously saved in the pickle.